docs/DeveloperGuide.md

```diff
@@ -1,35 +1,111 @@
----
-layout: page
-title: Developer Guide
----
-* Table of Contents
-{:toc}
-
+# TrackIt@NUS - Developer Guide
+
+By: `Team W13-4` Since: `Aug 2020` License: `MIT`
+
+## Table of Contents
+1. [**Introduction**](#introduction)
+2. [**Setting up**](#setup)
+3. [**Design**](#design)
+    1. [Architecture](#architecture)
+    2. [UI Component](#ui)
+    3. [Logic Component](#logic)
+    4. [Model Component](#model)
+    5. [Common Classes](#common)
+4. [**Implementation**](#implementation)
+    1. [Module Manager](#module-manager)
+    2. [Lesson Manager](#lesson-manager)
+    3. [Task Manager](#task-manager)
+    4. [Contact Manager](#contact-manager)
+    5. [Upcoming Tab](#upcoming-tab)
+    6. [App-Wide Search](#app-wide-search)
```

Comment:
we don't have the app-wide search anymore right? It will now be a contact tab?

Reply:
Ok I shall remove it then. There is already a Contact Manager section